<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainForm{
	background-color: #fbfcff;
}

QFrame#topLayout{
	border: 1px solid dodgerblue;
	border-radius: 10px;
}

QFrame#bottomLayout{
	border: 1px solid orangered;
	border-radius: 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="topLayout">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>لینک فایل</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="origianlLinkLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="getLinkButton">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Get Link</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bottomLayout">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>لینک نیم بها</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nimbahaLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyLinkButton">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
